registry.go

```diff
@@ -88,9 +90,21 @@ func (r *Registry) SetLogger(logger Logger) {
 	r.config.Log = logger
 }
 
+func (r *Registry) GetExport() map[string]Metric {
+	r.mutex.Lock()
+	defer r.mutex.Unlock()
```

Comment:
Yeah, this makes sense to me. However, what is the convention for checking if the lock is acquired? I wasn't aware you could do that in go. Or is the solution to just unlock immediately instead of defering?

Reply:
Ethan and I talked and we decided to remove the `mutex` code in the export getter.